src/scenes/Event/components/styles.js

```diff
@@ -1,11 +1,8 @@
 const styles = (theme) => ({
   oracleButton: {
+    marginTop: theme.padding.md.px,
+    background: theme.palette.primary.main,
     color: theme.palette.primary.contrastText,
```

Comment:
if you set `color='primary'` in the button itself, you shouldn't have to define the colors. look at how we use all the other buttons, eg. `Wallet/index.js L302`. when it comes to UI elements, try to be consistent with the same implementation as other places.